src/routes/safe/store/actions/updateSafeNonce.js

```diff
@@ -0,0 +1,14 @@
+// @flow
```

Comment:
this action is not needed, you can use `updateSafe` for this

Reply:
:grimacing: :man_facepalming:

Reply:
@mikheevm what about the `UPDATE_SAFE_THRESHOLD`? probably, if I'm going to add conditions to the `UPDATE_SAFE` reducer, I must move this over as well.

Reply:
(despite the latter does not have any condition. Even, thinking further, we can move all the safe-owners-related reducers into `UPDATE_SAFE` :thought_balloon:)

Reply:
ohh, I remember there was a particular issue why we created updateSafeThreshold action.. I need to remember why

Reply:
oh no I've mistaken

Reply:
it should be fine to move everything to update_safe

Reply:
Ok, I'm on it! :bomb:

Reply:
@mikheevm, refactored only for `THRESHOLD` and `NONCE`: https://github.com/gnosis/safe-react/pull/716/commits/372592981c277e22bf6ce0df20fc3cd6b7278558